stylesheets/all.scss

```diff
@@ -48,3 +48,10 @@ $screen-sm-min: 576px;
 .govuk-divider {
   border: solid 2px #01674e;
 }
+li {
```

Comment:
Are you sure you want to override the whole `li` selector?

Reply:
You should have a BEM class for the li tag and not style it directly.

Reply:
You should add a BEM style class name to the li if you want to style it.